File: loops/fizzBuzz.py
#Blaine Buckler, Fizz Buzz Python

num = 1
multiplier = 3  # Set any number for the multiplier
multiplierT = 5
while num < 51:
    if num % multiplier == 0 and num % multiplierT == 0: 
        print("FizzBuzz")
    elif num % multiplier == 0:  # Check if 'num' is a multiple of 'multiplier'
        print("Fizz")
    elif num % multiplierT == 0: 
        print("Buzz")
    else:
        print(num)
    num += 1

File: loops/fizzBuzz.c
//Blaine Buckler, Fizz Buzz C
#include <stdio.h>

int main() {
    int num = 1;          
    int multiplier = 3; 
    int multiplierT = 5; 

    while (num < 51) {
        if (num % multiplier == 0 && num % multiplierT == 0) {
            printf("FizzBuzz\n"); 
        } else if (num % multiplier == 0) {
            printf("Fizz\n"); 
        } else if (num % multiplierT == 0) {
            printf("Buzz\n"); 
        } else {
            printf("%d\n", num); 
        }
        num++; 
    }

    return 0; 
}
